bups/scheduler/systemd_user.py

```diff
@@ -0,0 +1,82 @@
+import ConfigParser
+import io
+import os
+import tempfile
+
+config_dir = os.path.join(os.path.expanduser("~"), ".config/systemd/user/")
+
+def is_available():
+    return os.path.exists(config_dir)
```

Comment:
Implemented this the right way now and checking for available executable. However, I've being testing on another machine, where anacron is also available, and ran into problems there. In gtk.py:497 the call to get_job() for anacron returns some good value, so the systemctl_user can't be called anymore. I'll create a bug report for that.